### PR TITLE
Homework hiding and encapsulation

### PR DESCRIPTION
**Pull request by Pticatos86**
Created class of company employees, added private attributes in created class, method __str__, decorators @property and @surname.setter

### CODE REVIEW

File: Args_kwargs_oop/homework_args_and_oop.py [hunk 1]
```diff
@@ -32,21 +32,82 @@ def delivery(self):
...
+                                  "food delivery, organization of birthdays and holidays, "
+                                  "service-'-50% I will pick it myself'",
+                                  '10.00 - 22.00', '+380968888888')
+    restaurant_two = MafiaCompany('restaurant_two', 'Dnipro',
+                                  "Volodymyr Monomakh street 2 K",
+                                  "Italian and Japanese cuisine",
+                                  "food delivery, organization of birthdays and holidays,"
+                                  "service-'-50% I will pick it myself'",
+                                  '10.00 - 22.00', '+380967777777')
+    print(f"Greeting to our guests - {restaurant_one.invite()}")
```
Comment (MarinaZelenska) [inline Args_kwargs_oop/homework_args_and_oop.py:109]: How I understand this code is not actual now

File: Args_kwargs_oop/homework_args_and_oop.py [hunk 2]
```diff
@@ -32,21 +32,82 @@ def delivery(self):
...
+        self._age = age
+        self.__salary = salary
+        self.__bonus = 10
+
+    def __str__(self):
+        return (f"{self.__class__.__name__} worker: works in {self._department} department, "
+                f"job title is {self._job_title}, name is {self._name}, surname is {self._surname}, "
+                f"age is {self._age}")
+
+    def __add__(self):
```
Comment (MarinaZelenska) [inline Args_kwargs_oop/homework_args_and_oop.py:50]: I don't understand why you add this magic method

**Review by MarinaZelenska — APPROVED** [review]

File: Args_kwargs_oop/homework_args_and_oop.py (hunk 1 above)
Comment (Pticatos86) [reply Args_kwargs_oop/homework_args_and_oop.py:109]: yes. this code was in previous homework

File: Args_kwargs_oop/homework_args_and_oop.py (hunk 2 above)
Comment (Pticatos86) [reply Args_kwargs_oop/homework_args_and_oop.py:50]: i added magic method __add__ ,used it  in function calculate salary when salary + bonus, so as not to create a new variable and return it in fuction

**Pull request merged**